rds_enhanced_monitoring/examples/generate_CWLogTestEvent.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SCRIPTPATH=$(cd `dirname $0` && pwd)
+
+ts=$(($(date +%s%N)/100000))
+tmp_CWLogEvent='''
+{
+  "awslogs": {
+    "data":"__b64data__"
+  }
+}
+'''
+
+tmp_data='''
+{
+  "logStream": "db-ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  "messageType": "DATA_MESSAGE",
+  "logEvents": [
+    {
+      "timestamp": __timestamp__,
+      "message": "__message__"
+    }
+  ],
+  "owner": "123456789000",
+  "subscriptionFilters": [
+    "dd-rdsenhanced-filter"
+  ],
+  "logGroup": "RDSOSMetrics"
+}
+'''
+
+tmp_message=$(sed -e 's/"/\\"/g' $SCRIPTPATH/tmp_message.json)
+
+tmp_CWLogRaw=${tmp_data/__message__/$tmp_message}
+tmp_CWLogB64=$(echo ${tmp_CWLogRaw/__timestamp__/$ts} | gzip | base64 -w0)
```

Comment:
`base64 -w0` is not valid on macOS

Reply:
I copied the changes you've made and updated my PR